agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -1512,6 +1512,7 @@ void ap_manager_thread::handle_hostapd_attached()
 {
     LOG(DEBUG) << "handling enabled hostapd";
 
+    ap_wlan_hal->read_supported_channels();
```

Comment:
Why do we read the supported channels here? we anyway read it from NL so why every time we attach to hostapd? This should be done only once.